frontend/src/lib/components/DateFilter/DateFilter.tsx

```diff
@@ -215,27 +214,25 @@ export function DateFilter({
         )
 
     return (
-        <LemonButtonWithDropdown
-            data-attr="date-filter"
-            id="daterange_selector"
```

Comment:
nit picking that we've lost `id="daterange_selector"` which could affect an action somewhere 🤷